scripts/xtensa-build-zephyr.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+# stop on most errors
+set -e
+
+SUPPORTED_PLATFORMS=(apl cnl icl tgl-h)
+BUILD_PATH=zephyr
```

Comment:
Relative or absolute path ? This should also be out of the SOF source directory too ? Probably worth a comment here.

Reply:
@lgirdwood by default `zephyr` relative to the current path. When using `-p` it can be either relative or absolute. I'll add a comment